app/webpacker/controllers/terms_and_conditions_controller.js

```diff
@@ -11,15 +12,23 @@ export default class extends ApplicationController {
   remove(event) {
     let confirmation = confirm(this.messageValue);
     if (confirmation) {
-      // remove shebang present in the enterprise/edit url
-      // because of the shebang rails cannot parse the url
-      // as it is not valid
       location.hash = "";
-
       this.stimulate(
         "EnterpriseEdit#remove_terms_and_conditions",
         event.target
       );
     }
   }
+
+  stopEvent(event) {
+    event.preventDefault();
+  }
+
+  add() {
+    let input = document.querySelector("#enterprise_terms_and_conditions");
```

Comment:
Why not using a `target`? Isn't the `#enterprise_terms_and_conditions` element in the terms_and_conditions_controller?

Reply:
I can't use `target` because the `#enterprise_terms_and_conditions` is in another instance of the t&c controller. This method is used by

```haml
%input.button.red{ type: 'button', value: t('js.admin.modals.continue'), "data-controller": "terms-and-conditions", "data-action": "click->help-modal#close click->terms-and-conditions#add" }
```
and this is also inside the t&c controller but in a different instance of it. It's controller comes from

```
# app/components/help_modal/help_modal_component.html.haml

%div{ id: @id, "data-controller": "help-modal", "data-action": "keyup@document->help-modal#closeIfEscapeKey" }
  .reveal-modal-bg.fade{ "data-help-modal-target": "background", "data-action": "click->help-modal#close" }
  
```